modules/internal/String.chpl

```diff
@@ -114,19 +114,21 @@ units for offsets or lengths:
  * codepoints
  * graphemes
 
-Most methods on the Chapel string type currently work with byte units by
-default. For example, :proc:`~string.length` returns the length in bytes and
-`int` values passed into :proc:`~string.this` are offsets in byte units.
+Most methods on the Chapel string type currently work with codepoint units by
+default. For example, :proc:`~string.length` returns the length in codepoints
```

Comment:
The comment here indicates string.length returns codepoints but it looks to me like the implementation returns the length in bytes.